tc/core/mapping_options_cpp_printer.cc

```diff
@@ -19,6 +19,8 @@
 
 namespace tc {
 
+MappingOptionsCppPrinter::~MappingOptionsCppPrinter() {}
```

Comment:
wouldn't `=default` in the class declaration work?

Reply:
it's pure, how would you write this:
```
~MappingOptionsCppPrinter() = 0, default;
```
?